docs/config.md

```diff
@@ -386,6 +386,19 @@ command will be executed directly. For example:
 ui.pager = ["sh", "-c", "diff-so-fancy | less -RFX"]
 ```
 
+Some formatters (like [`delta`](https://github.com/dandavison/delta)) require
+git style diffs for formatting. You can select this option on each invocation
+of `jj diff` by using the `--git` flag. You can also configure this style of
```

Comment:
> You can select this option on each invocation of `jj diff` by using the `--git` flag.

Since this section is about configuring a pager, I feel like we can skip that sentence and just describe how to configure `delta` in good way. What do you think?